modelName : "anymalModel"
friction : 1.0
loopRate : 400.0
maxTorque : 80.0
kpValue : 6000.0
kpValueZ : 6000.0
kdValue : 1800.0
kiValue : 0.0
kpSwingValue : 1000.0
kdSwingValue : 200.0
slackWeight : 0.1
initialReferencePose : [0.0, 0.0, 0.88, 0.0, 0.0, 0.0]